ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py

```diff
@@ -158,16 +167,31 @@ def addtional_check(e2e_yaml_file, upgrade_yaml_file, missed_jobs):
     args = sys.argv
     if len(args) < 3:
         raise Exception("Missing e2e config file or upgrade config file")
-    e2e_yaml = args[1]
-    upgrade_yaml = args[2]
-    
-    e2e_jobs = get_jobs(e2e_yaml)
-    upgrade_jobs = get_jobs(upgrade_yaml)
     
+    parser = argparse.ArgumentParser(description='Get missed upgrade jobs')
+    parser.add_argument("-e", "--e2e_yaml", type=str, 
+                    help='the path of e2e job file')
+    parser.add_argument("-u", "--upgrade_yaml",type=str,
+                    help='the path of upgrade job file')
     # upgrade mode, y means y version upgrade, z means z version upgrade
-    mode = args[3] if len(args) == 4 else 'y'
+    parser.add_argument("-m", "--mode", default="y", choices=["y", "z"])
+    # Only get e2e normal jobs which can be convert to upgrade
+    parser.add_argument("-j", "--jobs", default="no", choices=["yes", "no"])
```

Comment:
nice! btw if you set `--jobs` as `, action='store_true'` (without default="no", choices=["yes", "no"])
then just using `-j` by itself will count as True, while not using it as False

another idea may be adding `required=True` to `--e2e_yaml` so proper error is displayed if we forget to provide e2e job file

Reply:
yet another option is if no `--upgrade_yaml` is specified, automatically threat it as jobs == "yes", then we will not have to use jobs flag at all. but this is just my idea you can do something else as well :)